boa/src/vm/opcode.rs

```diff
@@ -471,6 +499,13 @@ pub enum Opcode {
     /// Stack: value, key, object **=>**
     SetPropertyByValue,
 
+    /// Defines a own property of an object by value.
+    ///
+    /// Operands:
+    ///
+    /// Stack: value, key, object **=>**
+    DefineOwnPropertyByValue,
```

Comment:
I think we should change the positions on this so its `object, key, value` so we don't have to call `swap2` (since its done a lot).